src/main/java/dev/meyi/bn/commands/BazaarNotifierCommand.java

```diff
@@ -175,4 +232,31 @@ public void processCommand(ICommandSender ics, String[] args) {
   public boolean canCommandSenderUseCommand(final ICommandSender sender) {
     return true;
   }
+
+
+  @Override
+  public List<String> addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
```

Comment:
This doesn't quite work. You should be checking the last argument and autocompleting based on what's already typed in:

"/bn fi" should autocomplete to "/bn find", not "/bn discord"

Reply:
I know But I haven found a good fix for it. It always starts at the beginning cyceling through

Reply:
https://stackoverflow.com/questions/12348673/string-arraylist-get-the-item-starting-with-this-string